rel/files/riak-admin

```diff
@@ -1048,11 +1048,11 @@ case "$1" in
 
         shift
 
-        MYPID=$$
+        RAND=$(($(($RANDOM % 1000)) + 1))
         NODE_NAME=${NAME_ARG#* }
         $ERTS_PATH/erl -noshell -noinput \
             -pa $RUNNER_LIB_DIR/basho-patches \
-            -hidden $NAME_PARAM riak_etop$MYPID$NAME_HOST $COOKIE_ARG \
+            -hidden $NAME_PARAM np_etop$RAND$NAME_HOST $COOKIE_ARG \
```

Comment:
Please document _why_ this is `np_etop` (it's mimicking node_package to avoid doubling the atom table usage).